src/server/action_handlers/actionkit-rsvp.js

```diff
@@ -73,13 +76,23 @@ export async function processAction(questionResponse, interactionStep, campaignC
           }
         }
         request.post({
-          'url': `${actionkitBaseUrl}/rest/v1/act/`,
+          'url': `${actionkitBaseUrl}/act/`,
           'form': userData
         }, function (err, httpResponse, body) {
           // TODO: should we save the action id somewhere?
-          if (err) {
-            console.error('actionkit event sign up failed', err, userData)
+          if (err || (body && body.error)) {
+            console.error('error: actionkit event sign up failed', err, userData, body)
           } else {
+            if (httpResponse.headers && httpResponse.headers.location) {
+              const actionId = httpResponse.headers.location.match(/action_id=([^&]+)/)
+              if (actionId) {
+                //save the action id of the rsvp back to the contact record
+                customFields['processed_event_action'] = actionId[1]
+                await r.knex('campaign_contact')
+                  .where('campaign_contact_id', campaignContactId)
```

Comment:
where does this `campaign_contact_id` come from? is this a new column, or should it be `campaign_contact.id`

Reply:
good catch!